apps/openchallenges/image-service/src/test/java/org/sagebionetworks/exception/BadRequestExceptionTest.java

```diff
@@ -0,0 +1,44 @@
+package org.sagebionetworks.openchallenges.image.service.exception;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class BadRequestExceptionTest {
+
+  @Test
+  public void BadRequestException_ConstructorTypeShouldMatch() {
```

Comment:
Use consistent function name format. This format is different from the first tests you have created, e.g. `getImage_ShouldReturnImage_WhenObjectKeyIsPassed()`. 

Also, function names should always start with a lowercase.

The same comments applies to other functions that are part of this PR.